docs/features/readonly-variables-and-parameters.md

```diff
@@ -0,0 +1,61 @@
+### High Level Language Proposal:  
+Allow 'readonly' to be used on local variables and on parameters. 
+
```

Comment:
As fields, const and readonly are different. I do not see a lot of observable difference in locals though. 

   const int x = 42;
   readonly int x = 42:

Do I use one or another?  (Ignoring that compiler will emit them the same way).